enterprise_access/apps/subsidy_access_policy/models.py

```diff
@@ -1275,15 +1275,15 @@ def can_allocate(self, number_of_learners, content_key, content_price_cents):
         # allocate won't exceed the remaining balance of the related subsidy.
         subsidy_balance = self.subsidy_balance()
         if self.content_would_exceed_limit(
-            total_allocated_and_spent_cents,
+            total_allocated_assignments_cents,
```

Comment:
this is the line which actually fixes the bug, the subsidy balance already accounts for spent (but not allocated)